conf/evaluator_upload.env

```diff
@@ -0,0 +1,5 @@
+POSTGRESQL_USER=ve_db_user_evaluator
+POSTGRESQL_PASSWORD=ve_db_user_evaluator_pwd
+EVALUATOR_TOPIC=vulnerability.evaluator.upload
+VMAAS_VULNERABILITIES_API=/api/v1/vulnerabilities
+PROMETHEUS_PORT=8085
```

Comment:
It looks like this PR is also including the split of evaluator into separate recalc and upload bits.  Should this be done in a different PR?

Reply:
Yes, it also splits evaluators to be similar to production. I've done it mainly for Grafana, because there are different charts for upload and recalc evaluator instances. It's true that it would be better to do it at least in two commints. On the other site, it's only about development stuffs and it's already done. I would merge it to safe time. But if you think it's important, I will split it into two commits tomorrow.

Reply:
Gotcha... makes sense.